ingest_development_scripts/testing_script.py

```diff
@@ -0,0 +1,43 @@
+from nucleus.sensorfusion import *
```

Comment:
This is close to being a real unit test but

A) The test data you use isn't being checked in
B) There are no automatic checks for expected data


Since this is a first pass, fine to skip for now, just letting you know this is VERY close!

Reply:
Will check this out!